UnifiedLog/Lib.py

```diff
@@ -1009,8 +1012,9 @@ def ProcessDataChunk(self, buffer, catalog, meta_chunk_index, debug_file_pos, lo
                                     logger.debug("fmt_str_v_offset highest bit set @ 0x{:X} ct={}".format(log_file_pos, ct))
                                 else:
                                     format_str, cache_a, cache_b = dsc_cache.ReadFmtStringAndEntriesFromVirtualOffset(fmt_str_v_offset)
-                            except:
+                            except (KeyError, IOError):
                                 logger.error('Failed to get DSC msg string @ 0x{:X} ct={}'.format(log_file_pos, ct))
+
```

Comment:
Here insert in the except block
 `format_str = '<compose failure [UUID]>'`

Reply:
ack I'll do the same at line 1000

Reply:
I'm not sure we should do that for the signpost_name. I do not know the default behavior on that from the log utility.